content/docs/integrations/confluence/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Confluence Search
+publishedDate: '2022-06-30T10:00:00.0Z'
+description: How to enable Confluence indexing for Roadie Search
+
+humanName: Confluence
+logoImage: '../../../assets/logos/confluence/confluence-icon.svg'
+integrationType: Integration
+---
+
+## Introduction
+
+Roadie supports integration with Atlassian Confluence wiki platform to enable cross application search functionality. Roadie indexes documents from Confluence so they are available for you to search within the Roadie search bar. 
+
+
+## Prerequisites
+
+You'll need an Atlassian account with an ability to create an API token and knowledge of the name of your Confluence organization.
+
+## Configuring Confluence Integration
+
+### Generate Atlassian API token
+
+Roadie uses Atlassian Confluence API to retrieve indexable information from the wiki platform. To be able to connect to your Confluence instance, we need an API token. 
+You can create an Atlassian token by navigating to your [**Atlassian** profile security page](https://id.atlassian.com/manage-profile/security) and Clicking 'Create and Manage API Tokens'.
+![Creating an Atlassian token](./confluence_api_token_generation.png)
+
+
+On the page navigated into you are able to generate API tokens which can be used to connect Roadie to your Confluence workspaces.
+
+
+### Construct Confluence Authentication Token to Work with Roadie
+
+The Confluence API uses Basic Authentication on their API to negotiate authentication. To create a compatible token you need to **base64 encode** your username and API token pair. To achieve that you can either use any of the online encoders or a command line tool. 
```

Comment:
I think we had a similar pattern with the jira (?) plugin and it didn't work well to ask the user to base64 encode things. No need to change this now but would be worth considering that when we need this we should ask for the user and password separately and do the base64 encoding in our logic. In this case it's a little cumbersome because we need to ask for the secret in one place and the user in a different one 🤔

Reply:
Good idea :+1: Changed, and modified the implementation to match as well